tests/rptest/tests/partition_movement_test.py

```diff
@@ -602,8 +602,7 @@ def test_deletion_stops_move(self, num_to_upgrade):
 
         # create a single topic with replication factor of 1
         topic = 'test-topic'
-        rpk = RpkTool(self.redpanda)
-        rpk.create_topic(topic, 1, 1)
+        self.create_topic(topic, 1, 1)
```

Comment:
would using `DefaultClient` help ?

Reply:
DefaultClient uses java API it spinning a JVM just to create a topic is too slow